bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Control.java

```diff
@@ -1445,6 +1445,7 @@ public boolean forceFocus () {
 	Decorations shell = menuShell ();
 	shell.setSavedFocus (this);
 	if (!isEnabled () || !isVisible () || !isActive ()) return false;
+	if (display.getActiveShell() != getShell() && !getDisplay().activateShellOnForceFocus) return false;
```

Comment:
Is `getDisplay().` intentionally used instead of `display.` here?

Reply:
No, the `getDisplay()` was not by intention. I've replaced it with `display` in the new pull request.